components/job-orchestration/job_orchestration/garbage_collector/utils.py

```
@@ -102,8 +102,8 @@ class DeletionCandidatesBuffer:
 
     def __init__(self, recovery_file_path: pathlib.Path):
         self._recovery_file_path: pathlib.Path = recovery_file_path
-        self._candidates: Set[str] = set()
-        self._candidates_to_persist: List[str] = list()
+        self._candidates: set[str] = set()
+        self._candidates_to_persist: list[str] = list()
```

Ruff (0.14.5)

Unnecessary `list()` call (rewrite as a literal)

Rewrite as a literal

(C408)